test

```diff
@@ -1,35 +1,28 @@
-#!/usr/bin/env bash
+#!/bin/bash -e
 
 set -eu
 
-source ./build
-
 SRC=$(find . -name '*.go' \
 	-not -path "./vendor/*")
 
-PKG=$(cd gopath/src/${REPO_PATH}; go list ./... | \
-	grep --invert-match vendor)
-
-# https://github.com/golang/go/issues/15067
-PKG_VET=$(cd gopath/src/${REPO_PATH}; go list ./... | \
-	grep --invert-match vendor | \
-	grep --invert-match internal/log)
+PKGS=$(go list ./... | grep -v /vendor)
+FORMATTABLE=$(ls -d */ | grep -v -E '(vendor/|examples/)')
 
 echo "Checking gofix..."
 go tool fix -diff $SRC
 
 echo "Checking gofmt..."
```

Comment:
were you going to add an issue or TODO to add `golint` checking? Its the only one missing